### ISSUE TITLE
lack of deeper content as a limitation versus a design choice

### ISSUE THREAD

**Issue by corybrunson** [TrevorFrench/R-for-Data-Analysis#23]
@tomsing1 brought this up in the review thread and i thought it would be worth asking about directly. I agree that the book is conspicuously lacking in real-world examples, as well as in theoretical background or discussions of good practice, for example, in comparison to other book-length resources. However, this struck me more as a design choice than as a limitation; though it was not explicitly mentioned in the README or the JOSE paper. If it is intentional—for example, if the resource is intended for users with background or other resources for these elements—then i think it would be good to explain this in the documentation, and to link to complementary resources there. (But i'm glad to be rejoined on this.)

_Part of [this JOSE review](https://github.com/openjournals/jose-reviews/issues/202)._

**Comment by corybrunson** [comment]
Actually, part of this has already been addressed in response to other issues! But i'm still interested in hearing any thoughts.

**Comment by TrevorFrench** [comment]
@corybrunson yes, this was a choice! I updated the statement of need to include the following:

"Furthermore, the contents of R for Data Analysis (French, 2022) are centered around the idea of the "process of data analysis" broadly applied to any discipline. This differs from other high-quality resources, such as "R for Reproducible Scientific Analysis" (Zimmerman et al., 2019) which teaches similar topics in the context of the scientific process."
